daemon/graphdriver/devmapper/mount.go

```diff
@@ -59,6 +59,7 @@ func ProbeFsType(device string) (string, error) {
 	buffer := make([]byte, maxLen)
 	l, err := file.Read(buffer)
 	if err != nil {
+		file.Close()
```

Comment:
use defer file.Close() after the previous err check is also ok, however,shall we use current modify to keep the coding style accordnate to the original?